src/components/datalist-item/DatalistItem.js

```diff
@@ -46,14 +47,14 @@ class DatalistItem extends React.Component<Props> {
     id: string;
 
     render() {
-        const { children, className, isActive, ...rest } = this.props;
+        const { children, className, isActive, name, ...rest } = this.props;
         const classes = classNames('datalist-item', { 'is-active': isActive }, className);
         const itemProps = omit(rest, ['closeDropdown', 'setActiveItemID']);
 
         // required aria props are added dynamically
         /* eslint-disable jsx-a11y/role-has-required-aria-props */
         return (
-            <li {...itemProps} className={classes} id={this.id} role="option">
+            <li {...itemProps} className={classes} id={this.id} role="option" title={name}>
```

Comment:
title cannot be a React.Node, only string

Reply:
Turns out the flowtype for name should be string and not React.Node. Will be using name prop instead of title prop